storage.js

```diff
@@ -0,0 +1,37 @@
+const store = {
+	currentStore: null,
+
+	init() {
+		//get items from storage
+		//add listeners
+	},
+
+	get(params) {
+		// use async, await here because the below returns a promise
+		// empty params, return all items from storage
+		return browser.storage.local.get();
+	},
+
+	add(item) {
+	 	this.set(item);	
+	},
+
+	update(item) {
+		this.set(item);
+	},
+
+	set(item) {
+		// item ? update : add;
+		browser.storage.local.set({ item });
```

Comment:
This won't work, it will set an item called item.
This is shorthands confusing you (don't worry it gets me too).
`{item}` is short for `{item: item}`